contracts/CollectionFoundation.sol

```diff
@@ -0,0 +1,47 @@
+pragma solidity ^0.8.0;
+
+import "./freeportParts/MetaTxContext.sol";
+import "./Collection.sol";
+
+/** This is a contract for creating standalone contracts (collections) for users.
```

Comment:
Is it the same "Foundation" for all users, or one personal per user?

Reply:
Yeah, it's better to call it Factory and it manages all collections (for all users).